WordPress/Classes/ViewRelated/Me/MeViewController.m

```diff
@@ -44,6 +45,11 @@ @interface MeViewController () <UITableViewDataSource, UITableViewDelegate, UIAc
 
 @implementation MeViewController
 
++ (UIViewController *)viewControllerWithRestorationIdentifierPath:(NSArray *)identifierComponents coder:(NSCoder *)coder
+{
+    return [[self alloc] init];
```

Comment:
Is there any testing path causing this line to be executed?  I placed a breakpoint here and never got it to execute.